apps/www/lib/context/MeContext.tsx

```diff
@@ -79,6 +86,30 @@ type MeContextValues = {
   hasAccess: boolean
   isEditor: boolean
   isClimateLabMember: boolean
+  trialStatus?: TrialStatusType
+}
+
+const getTrialStatus = (me?: MeObjectType | undefined): TrialStatusType => {
```

Comment:
this function could/should maybe come from the API? @lucianak

Reply:
so that would be my suggestion